scalapact-http4s-0-18-0/src/main/scala/com/itv/scalapact/shared/http/PactStubService.scala

```diff
@@ -27,14 +27,14 @@ object PactStubService {
     println(("Starting ScalaPact Stubber on: http://" + config.giveHost + ":" + config.givePort.toString).white.bold)
     println(("Strict matching mode: " + config.giveStrictMode.toString).white.bold)
 
-    runServer(interactionManager, nThreads, sslContextName)(pactReader, pactWriter, sslContextMap)(config).awaitShutdown()
+    runServer(interactionManager, nThreads, sslContextName, config.givePort)(pactReader, pactWriter, sslContextMap)(config).awaitShutdown()
   }
 
   implicit class BlazeBuilderPimper(blazeBuilder: BlazeBuilder[IO]) {
     def withOptionalSsl(sslContext: Option[SSLContext]): BlazeBuilder[IO] = sslContext.fold(blazeBuilder)(ssl => blazeBuilder.withSSLContext(ssl))
   }
 
-  def runServer(interactionManager: IInteractionManager, connectionPoolSize: Int, sslContextName: Option[String])
+  def runServer(interactionManager: IInteractionManager, connectionPoolSize: Int, sslContextName: Option[String], port: Int)
```

Comment:
Why adding the `port` parameter that's not used in the function? It might confuse the client.

Reply:
I've had another look. There is something fishy going on here but I've made it look like the other for now so that at least the behaviour will be consistent. Will review outside of this PR.